.github/workflows/trivy-scheduled-scan.yaml

```diff
@@ -0,0 +1,76 @@
+name: Trivy Scheduled Scans
+
+on:
+  # Scan schedule is the same as codeql-analysis job.
```

Comment:
💭 if this cron is both for the master branch security tab & the backbranch slack message, maybe we want to switch it to the other cron we have? so that we get messages in the weekday morning? `0 14 * * 1`?

Reply:
That sounds good.